src/lib/files/MaestroConfig.ts

```diff
@@ -18,7 +18,7 @@ export interface MaestroConfig {
     excludeFilters?: string[];
     addFrameworkFiles?: boolean;
     stripParamTypes?: boolean;
-    paramStripExceptions?: string;
+    paramStripExceptions?: string[];
```

Comment:
Changed this to a string array, because it defaults to an array here: https://github.com/georgejecook/maestro-roku-bsc-plugin/blob/45e06abd1268156563e3d420cdef10291b31637c/src/plugin.ts#L132 . Not sure if that's ok, but it seemed like an oversight that it could be both a string OR an array.

Reply:
this is correct. sorry for slow response.